Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSuggestionsViewController.swift

```diff
@@ -38,27 +37,34 @@ open class _ChatMessageComposerSuggestionsViewController<ExtraData: ExtraDataTyp
     /// the bottomAnchorView as soon as we compute the height of the
     /// contentSize of the nested collectionView
     public var bottomAnchorView: UIView?
-
+    
+    /// A closure to observer when an item is selected.
     public var didSelectItemAt: ((Int) -> Void)?
-
+    
+    /// Property to check if the suggestions view controller is currently presented.
     public var isPresented: Bool {
         view.superview != nil
     }
-
-    private var collectionViewHeightObserver: NSKeyValueObservation?
-
-    // MARK: - Subviews
-
+    
+    /// The collection view of the commands.
     open private(set) lazy var collectionView = uiConfig
         .messageComposer
         .suggestionsCollectionView
         .init(layout: uiConfig.messageComposer.suggestionsCollectionViewLayout.init())
         .withoutAutoresizingMaskConstraints
-
+    
+    /// The container view where collectionView is embedded.
     open private(set) lazy var containerView: UIView = UIView().withoutAutoresizingMaskConstraints
 
-    // MARK: - Overrides
-
+    // Height for suggestion cell, this value should never be 0
+    // otherwise it causes loop for height of this controller and as a result this controller height will be 0 as well.
+    // Note: This value can be 1, it's just for purpose of 1 cell being visible.
```

Comment:
You could add the documentation that this is for purpose of seeing just 1 cell to adjust height and not looping to 0 according to our Slack conversation.

Reply:
The document that is already present seems enough? Or not? 🤔

Reply:
Ahh in that case it's okay :D Sorry I was confused from our Slack conversation... This is good actually 👍